modules/ROOT/pages/exportpdf.adoc

```diff
@@ -0,0 +1,63 @@
+= Export to PDF plugin
+:navtitle: Export to PDF
+:description: The Export to PDF feature lets you generate a PDF file directly from the editor.
+:description_short: Generate a PDF file directly from the editor.
+:keywords: plugin, exportpdf, export to pdf
+:pluginname: Export to PDF
+// values for <tier>
+// opensource plugin = tieroneplan
+// premium plugin    = tiertwoplan or tierthreeplan or enterpriseplan
```

Comment:
@ShiridiGandham Can you confirm what we want to set this to.

Reply:
The approved Export to word plugin docs also has this open.

Reply:
```suggestion
```